src/behavior-property-observer.js

```diff
@@ -92,3 +117,51 @@ export class BehaviorPropertyObserver {
     this.removeSubscriber(context, callable);
   }
 }
+
+const numCons = Number;
+const dateCons = Date;
+const _isFinite = isFinite;
+const _isNaN = isNaN;
+
+export const coerces = {
+  none(a) {
+    return a;
+  },
+  number(a) {
+    var val = numCons(a);
+    return !_isNaN(val) && _isFinite(val) ? val : 0;
+  },
+  string(a) {
+    return '' + a;
+  },
+  boolean(a) {
+    return !!a;
+  },
+  date(a) {
+    return new dateCons(a);
+  }
+};
+
+/**@type {Map<Function, string>} */
+export const classCoerceMap = new Map([
```

Comment:
You are right about this, could have inlined type def.
But the purpose of this variable is to hold a map between Classes and their coerce type (as string) in real look up table of `coerces` for later use. So the type would be `Map<Function, string>`